kitsune/community/templates/community/top_contributors_react.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% set styles = ('common', 'community-new',) %}
+{% if area == 'questions' %}
+    {% set scripts = ('community-new-questions',) %}
+{% elif area == 'l10n' %}
+    {% set scripts = ('community-new-l10n',) %}
+{% endif %}
+
+{% if area == 'questions' %}
+    {% set title = _('Top Contributors - Support Forum') %}
+{% elif area == 'kb' %}
+    {% set title = _('Top Contributors - Knowledge Base') %}
+{% elif area == 'l10n' %}
+    {% set title = _('Top Contributors - Localization') %}
+{% elif area == 'army-of-awesome' %}
+    {% set title = _('Top Contributors - Army of Awesome') %}
+{% endif %}
+{% set crumbs = [(url('community.home'), _('Community Hub')),
+                 (None, title)] %}
+
+{% block content %}
+    <script type="application/json" name="contributor-data">{{ contributors_json|safe }}</script>
+    <script type="applicatoin/json" name="locale-data">{{ locales_json|safe }}</script>
```

Comment:
Inline *Javascript* scripts is a problem with CSP, for injection reasons. Other kinds of script tags, like JSON data, are fine, since nothing automatically executes them. xWe could do data tags, but it seemed better to do script tags because a) we can note the type of the data, and b) it is a lot of data that I don't really want to jam in as an attribute on a tag.

Also, I notice there is a typo here, "applicatoin".

It could work either way though, so I could change it to be data attributes, if you still think it should be that way.